### PR TITLE
FireSim 1.6.0 Release (Dev -> Master PR)

### PR DESCRIPTION
**Pull request by davidbiancolin**
See firesim/firesim#262.

### CODE REVIEW

File: Makefrag [hunk 1]
```diff
@@ -1,15 +1,22 @@
 ROCKETCHIP_DIR=$(base_dir)/rocket-chip
 
-SBT ?= java -Xmx2G -Xss8M -XX:MaxPermSize=256M -jar $(ROCKETCHIP_DIR)/sbt-launch.jar ++2.12.4
+SCALA_VERSION := 2.12
+SCALA_PATH := 4
+SBT ?= java -Xmx2G -Xss8M -XX:MaxPermSize=256M -jar $(ROCKETCHIP_DIR)/sbt-launch.jar ++$(SCALA_VERSION).$(SCALA_PATCH)
 
 lookup_scala_srcs = $(shell find $(1)/ -iname "*.scala" 2> /dev/null)
 
+space :=
```
Comment (alonamid) [inline Makefrag]: Can we add a comment to explain this?

File: vsim/Makefile [hunk 2]
```diff
@@ -16,16 +16,10 @@ debug: $(simv_debug)
...
-
 sim_vsrcs = \
 	$(build_dir)/$(PROJECT).$(MODEL).$(CONFIG).v \
-	$(rocketchip_vsrc_dir)/TestDriver.v \
-	$(rocketchip_vsrc_dir)/AsyncResetReg.v \
-	$(rocketchip_vsrc_dir)/plusarg_reader.v \
-	$(icenet_vsrcs) $(testchip_vsrcs)
 
-sim_csrcs = $(icenet_csrcs) $(testchip_csrcs)
+sim_csrcs =
```
Comment (alonamid) [inline vsim/Makefile]: Also empty?

**Review by alonamid — APPROVED** [review]
Generally seems ok, other than a couple of comments about the Makefiles

File: Makefrag (hunk 1 above)
Comment (davidbiancolin) [reply Makefrag]: @zhemao

File: vsim/Makefile (hunk 2 above)
Comment (davidbiancolin) [reply vsim/Makefile]: @zhemao

**Comment by davidbiancolin** [comment]
> Generally seems ok, other than a couple of comments about the Makefiles

I tagged Howie on those questions, but given our plan to transition to Chipyard, i'm just going to merge this once everything else is ready.

**Pull request merged**